crates/host/src/wasmbus/mod.rs

```diff
@@ -644,6 +644,38 @@ fn linkdef_hash(
     hex::encode_upper(hash.finalize())
 }
 
+#[instrument(skip(jetstream))]
+async fn create_lattice_metadata_bucket(
+    jetstream: &JetstreamContext,
+    bucket: &str,
+) -> anyhow::Result<()> {
+    match jetstream
+        .create_key_value(kv::Config {
+            bucket: bucket.to_string(),
+            ..Default::default()
+        })
+        .await
+    {
+        Ok(_) => {
+            info!("created lattice metadata bucket {bucket} with 1 replica");
+            Ok(())
+        }
+        Err(err) => {
+            // Ignore the error if the bucket was pre-provisioned
+            // This is a terrible hack, but the error we get from the NATS client is a nested
+            // dyn Error. The NATS server will return an error containing the code
+            // 10058 if the underlying stream already exists with different configuration,
+            // which is the case we care about
+            if err.to_string().contains("10058") {
+                info!("lattice metadata bucket {bucket} already exists. Skipping creation.");
+                Ok(())
+            } else {
+                Err(anyhow!(err).context("failed to create lattice metadata bucket {bucket}"))
```

Comment:
`{bucket}` is not interpolated here - you should be using `format!` macro.
Also, please surround the variable with backticks.